enterprise/signals.py

```diff
@@ -55,8 +56,9 @@ def handle_user_post_save(sender, **kwargs):  # pylint: disable=unused-argument
         # actually exist in the system; in such a case, the enrollment for each such
         # course is finalized when the user registers with the OpenEdX platform.
         enrollment.complete_enrollment()
-        EnterpriseCourseEnrollment.objects.get_or_create(
+        EnterpriseCourseEnrollment.objects.create(
```

Comment:
Why change this? I think we should revert.

Reply:
@douglashall I changed it because the `EnterpriseCustomerUser` associated with this course enrollment is only created [several lines above](https://github.com/edx/edx-enterprise/blob/7ad0b8bced8c033805d678e1c4413e0819236c3f/enterprise/signals.py#L50-L53), so an `EnterpriseCourseEnrollment` linked to the new `EnterpriseCustomerUser` instance cannot possibly exist at this stage in the code.